Web/Types.hs

```diff
@@ -4,6 +4,21 @@ import IHP.Prelude
 import IHP.ModelSupport
 import Generated.Types
 
+-- Custom types
+
+{-| The result of fetching Landing page with all its paragraphs.
+
+@see `fetchLandingPageWithRecords`
+-}
+data LandingPageWithRecords = LandingPageWithRecords
```

Comment:
Create a new data type to have a record with all the fetched data

Reply:
👍 

But why these long field names? Wouldn't `landingPage` work as well instead of `landingPageWithRecordsLandingPage`?